app/src/main/java/com/kickstarter/ui/extensions/ImageViewExt.kt

```diff
@@ -106,12 +109,11 @@ fun ImageView.loadImage(url: String?, context: Context, imageViewPlaceholder: Ap
                             isFirstResource: Boolean
                         ): Boolean {
                             targetView.setImageDrawable(null)
-                            imageViewPlaceholder?.setImageDrawable(null)
+                            imageZoomablePlaceholder?.setImageDrawable(null)
                             return isFirstResource
                         }
                     })
-                    .placeholder(ColorDrawable(Color.TRANSPARENT))
-                    .diskCacheStrategy(DiskCacheStrategy.ALL)
+                    .diskCacheStrategy(DiskCacheStrategy.AUTOMATIC)
```

Comment:
DiskCacheStrategy.AUTOMATIC intelligently chooses a cache strategy based on the resource